### ISSUE TITLE
volspotconnect2 doesn't connect anymore

### ISSUE THREAD

**Issue by raffomartini** [balbuze/volumio-plugins#130]
Hi, 

I've been using this plugin for months, today it suddenly stopped working:
I can see Volumio from the spotify app, but it doesn't connect.

Where can I see the logs and figure out what's going on?

thank you

**Comment by EchoZulu** [comment]
@raffomartini Hi, did you solved problem? (i have the same problem)

**Comment by serwinch** [comment]
Me too.
I notice that it works with the PC application, with the IPad application, but no more with the Android application.

**Comment by raffomartini** [comment]
TL;DR;  it started working on its own after few hours, I suspect some issue with the Spoty API on their side but I have no way to confirm it.

Afraid I never really went to the bottom of it..
Chronology
- volspotconnect stop working.
- restart volumio -> nothing
- update the plugin and reload -> nothing
- install volspotconnect2 and reload -> nothing
- log the bug to further investigate
- try again after few hours -> working
- marked bug as closed

Conclusion: as it recovered "on its own" I suspect it was something on Spoty API but - not knowing where to look for clues - it is only a wild guess.

**Comment by balbuze** [comment]
This is a known issue due to a change in spotify. Install the plugin from my repo or wait for an update of the official version

**Comment by raffomartini** [comment]
Thanks balbuze.
It stopped working again today, your update is SUPEEER timely.
I have it working now.

## Here's how I solved it: 

Link to the plugin:
https://github.com/balbuze/volumio-plugins/raw/master/plugins/music_service/volspotconnect2/volspotconnect2.zip
(source https://github.com/balbuze/volumio-plugins/issues/135)

### How to do manual installation now that the option is not in the GUI anymore:
Login to volumio via ssh `ssh volumio.local -l volumio` (default password is volumio)
```
cd /data/INTERNAL/
wget https://github.com/balbuze/volumio-plugins/raw/master/plugins/music_service/volspotconnect2/volspotconnect2.zip
miniunzip volspotconnect2.zip 
volumio plugin install
```
(inferred from https://github.com/balbuze/volumio-plugins/issues/131)
